dbms/src/Functions/CMakeLists.txt

```diff
@@ -20,8 +20,7 @@ target_link_libraries(clickhouse_functions
         ${METROHASH_LIBRARIES}
         murmurhash
         ${BASE64_LIBRARY}
-        ${OPENSSL_CRYPTO_LIBRARY}
-        ${LZ4_LIBRARY})
```

Comment:
`LZ4_LIBRARY` is not needed here due to `USE_XXHASH` added below. It came from https://github.com/yandex/ClickHouse/pull/3905/files#diff-6ad97c74dd7af3de638524e1b08e1093R25.